scripts/mainnet/redeploy_subscription.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+
+from ape import project
+
+from deployment.constants import CONSTRUCTOR_PARAMS_DIR
+from deployment.params import Deployer
+
+VERIFY = False
+CONSTRUCTOR_PARAMS_FILEPATH = CONSTRUCTOR_PARAMS_DIR / "mainnet" / "redeploy-marlin.yml"
```

Comment:
Perhaps the constructor params file name part of this path  i.e. "redeploy-marlin.yml" can be  passed as a parameter instead to this script? Seems like this script is meant to be used with any subscription constructor params file, each with their own parameter customizations.

(cc @cygnusv , @vzotova )